napalm_ios/ios.py

```diff
@@ -581,10 +581,14 @@ def get_optics(self):
         for optics_entry in split_output.splitlines():
             # Example, Te1/0/1      34.6       3.29      -2.0      -3.5
             try:
-                int_brief, temperature, voltage, output_power, input_power = optics_entry.split()
+                split_list = optics_entry.split()
             except ValueError:
                 return {}
 
+            int_brief = split_list[0]
+            output_power = split_list[3]
+            input_power = split_list[4]
+
```

Comment:
Yes, as in some cases split gives more items than usually. If tx/rx power has crossed  warning or alarm threshold there are one or two minus or plus signs in the output. See newly added unit test as example. In this case you have to survive from variable number of list items.